apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/EphemeralTransaction.cs

```diff
@@ -133,13 +133,14 @@ async Task<int> Impl(SpannerTransaction transaction)
             }
         }
 
-        public Task<ReliableStreamReader> ExecuteReadOrQueryAsync(ReadOrQueryRequest request, CancellationToken cancellationToken, int timeoutSeconds /* ignored */)
+        public Task<ReliableStreamReader> ExecuteReadOrQueryAsync(ReadOrQueryRequest request, CancellationToken cancellationToken, int timeoutSeconds /* ignored */, bool isDmlReturn = false)
```

Comment:
In my own equivalent branch, I ended up putting the command type (as SpannerCommandType) in the EphemeralTransaction constructor. That meant I didn't need to modify any of the other transaction types.